odfdom/src/main/java/module-info.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2022 The Apache Software Foundation.
```

Comment:
There is a copy/paste error here:
The Copyright does not belong to ASF but is likely belonging to "The Document Foundation" or you have the copyright.
My thanks to Andy Seabourne,  for [pointing this out](https://issues.apache.org/jira/browse/JENA-2253?page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel&focusedCommentId=17476653#comment-17476653).